targets/UnrealMarketplacePlugin/make.js

```diff
@@ -42,7 +63,11 @@ exports.makeCombinedAPI = function (apis, sourceDir, baseApiOutputDir) {
             hasClientOptions: authMechanisms.includes("SessionTicket"),
             hasServerOptions: authMechanisms.includes("SecretKey"),
             sdkVersion: sdkGlobals.sdkVersion,
-            ueTargetVersion: ueTargetVersion,
+			ueTargetVersionMajor: ueTargetVersion.major,
+			ueTargetVersionMinor: ueTargetVersion.minor,
+			ueTargetVersionPatch: ueTargetVersion.patch,
```

Comment:
I don't think we should send these 3 into the templates.
They're not needed now, and the two below are, so let's just stick with what's used.